.github/workflows/build-assets.yml

```diff
@@ -120,14 +120,14 @@ jobs:
           - os: ubuntu-20.04
             name: LN
             ln: true
-          - os: centos-8
-            name: LN
-            cmake_options_extra: ""
-            ln: true
```

Comment:
Are we 100% sure we can drop CentOS 8?

Reply:
centos-8 and rocky8 will generate the same rpm package name with "el8" marker since both are in line with RedHat Enterprices 8